src/components/textfieldButtons/js/TextfieldButtons.js

```diff
@@ -0,0 +1,166 @@
+/*
+Copyright 2013 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
+*/
+
+var fluid_2_0_0 = fluid_2_0_0 || {};
+
+(function ($, fluid) {
+    "use strict";
+
+    /********************
+     * Textfield Buttons *
+     ********************/
+
+    fluid.defaults("fluid.textfieldButtons", {
+        gradeNames: ["fluid.viewComponent"],
+        components: {
+            textfield: {
+                type: "fluid.textfieldButtons.textfield",
+                container: "{textfieldButtons}.dom.textfield",
+                options: {
+                    model: "{textfieldButtons}.model",
+                    range: "{textfieldButtons}.options.range"
+                }
+            },
+            increaseButton: {
+                type: "fluid.stepper.button",
+                container: "{textfieldButtons}.dom.increaseButton",
+                options: {
+                    model: "{fluid.textfieldButtons}.model",
+                    range: "{fluid.textfieldButtons}.options.range",
+                    buttonOptions: "{fluid.textfieldButtons}.options.buttonOptions",
+                    incrementCoefficient: 1
+                }
+            },
+            decreaseButton: {
+                type: "fluid.stepper.button",
+                container: "{textfieldButtons}.dom.decreaseButton",
+                options: {
+                    model: "{fluid.textfieldButtons}.model",
+                    range: "{fluid.textfieldButtons}.options.range",
+                    buttonOptions: "{fluid.textfieldButtons}.options.buttonOptions",
+                    incrementCoefficient: -1
+                }
+            }
+        },
+        selectors: {
+            textfield: ".flc-textfieldButtons-field",
+            increaseButton: ".fl-increase-button",
+            decreaseButton: ".fl-decrease-button"
+        },
+        model: {
+            value: null
+        },
+        modelRelay: {
+            target: "value",
+            singleTransform: {
+                type: "fluid.transforms.limitRange",
+                input: "{that}.model.value",
+                min: "{that}.options.range.min",
+                max: "{that}.options.range.max"
+            }
+        },
+        range: {
+            min: 0,
+            max: 100
+        }
+    });
+
+    fluid.defaults("fluid.textfieldButtons.textfield", {
```

Comment:
This component seems a copy-paste of ```fluid.textfieldSlider.textfield``` at https://github.com/fluid-project/infusion/blob/master/src/components/textfieldSlider/js/TextfieldSlider.js#L95

Please use the original component directly. Every time when you do a copy and paste, that is a warning signal. Thinking deeper in those cases to see if you can avoid the code repetition by creating a common utility function or creating a common grade component or refactoring the code to reuse common parts.